### ISSUE TITLE
Cirrus-Ci Job failes due to timeout

### ISSUE THREAD

**Issue by tibeer** [osism-archive/openstack-ironic-images#35]
**build_ubuntu2004_amd64** is running into a timeout of the job. Found by pull request: https://github.com/osism/openstack-ironic-images/pull/34
How can this be debugged?

**Comment by muellerbe** [comment]
it is interessting, the cirrus output is empty for this job

you can run the command in job in your branch for debugging, maybe this helps

**Comment by muellerbe** [comment]
ah, there is a log link
https://api.cirrus-ci.com/v1/task/6051508503445504/logs/build_ubuntu2004_amd64.log
I cloned this branch and build the image, it works for me ;) maybe a timing problem in cirrus?

**Comment by muellerbe** [comment]
packer build ubuntu-20.04-amd64.json
[1;32mqemu: output will be in this color.[0m

[1;32m==> qemu: Retrieving ISO[0m
[1;32m==> qemu: Trying http://cdimage.ubuntu.com/ubuntu-legacy-server/releases/20.04/release/ubuntu-20.04.1-legacy-server-amd64.iso[0m
[1;32m==> qemu: Trying http://cdimage.ubuntu.com/ubuntu-legacy-server/releases/20.04/release/ubuntu-20.04.1-legacy-server-amd64.iso?checksum=sha256%3Af11bda2f2caed8f420802b59f382c25160b114ccc665dbac9c5046e7fceaced2[0m
[1;32m==> qemu: http://cdimage.ubuntu.com/ubuntu-legacy-server/releases/20.04/release/ubuntu-20.04.1-legacy-server-amd64.iso?checksum=sha256%3Af11bda2f2caed8f420802b59f382c25160b114ccc665dbac9c5046e7fceaced2 => /tmp/cirrus-ci-build/.cache/packer/48e4ec4daa32571605576c5566f486133ecc271f.iso[0m
[1;32m==> qemu: Starting HTTP server on port 8988[0m
[1;32m==> qemu: Found port for communicator (SSH, WinRM, etc): 4381.[0m
[1;32m==> qemu: Looking for available port between 5900 and 6000 on 127.0.0.1[0m
[1;32m==> qemu: Starting VM, booting from CD-ROM[0m
[0;32m    qemu: The VM will be run headless, without a GUI. If you want to
    qemu: view the screen of the VM, connect via VNC without a password to
    qemu: vnc://127.0.0.1:5959[0m
[1;32m==> qemu: Waiting 10s for boot...[0m
[1;32m==> qemu: Connecting to VM via VNC (127.0.0.1:5959)[0m
[1;32m==> qemu: Typing the boot command over VNC...[0m
[0;32m    qemu: Not using a NetBridge -- skipping StepWaitGuestAddress[0m
[1;32m==> qemu: Using SSH communicator to connect: 127.0.0.1[0m
[1;32m==> qemu: Waiting for SSH to become available...[0m

Timed out!

**Comment by berendt** [comment]
@tibeer Please be more active in bringing issues to an end in the future.

https://cirrus-ci.org/guide/custom-vms/

```
Make sure that your source image already has a [necessary license](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances#enablenestedvirt). Otherwise, nested virtualization won't work.
```

Nested Virtualization was previously included in the Free Tier at Cirrus CI. Doesn't seem to work now, unfortunately. Therefore, QEMU is then used and it takes a very long time.

I'm kicking out Cirrus CI for now and building the image manually until we find a better solution.
